website/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
```

Comment:
tested also on local and working fine

fails when folders are not present
fails when run in wrong folder (as expected)